src/components/calcite-tab-title/calcite-tab-title.scss

```diff
@@ -9,15 +9,11 @@ $tab-margin: 1.25rem;
 }
 
 // only ie11 and chrome will run this, others have logical properties
-:host-context([dir="rtl"]) {
```

Comment:
hard to check this one as tabs are completely broken in ie right now 😆 ... 😭

Reply:
I've updated this to use a class as well so it will work when nested deeply.

Reply:
nice!